toolkit/tools/pkg/imagecustomizerlib/extractpartitions.go

```diff
@@ -0,0 +1,87 @@
+package imagecustomizerlib
+
+import (
+	"fmt"
```

Comment:
Remember to run `go fmt`.

Reply:
Yes, done already as mentioned in PR description